kustomize/commands/edit/add/addreplacement.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2023 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package add
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/kustomize/api/konfig"
+	"sigs.k8s.io/kustomize/api/types"
+	"sigs.k8s.io/kustomize/kustomize/v5/commands/internal/kustfile"
+	"sigs.k8s.io/kustomize/kyaml/filesys"
+)
+
+type addReplacementOptions struct {
+	Replacement types.ReplacementField
+}
+
+func newCmdAddReplacement(fSys filesys.FileSystem) *cobra.Command {
+	var o addReplacementOptions
+	cmd := &cobra.Command{
+		Use:   "replacement",
+		Short: "add an item to replacement field",
+		Long: `this command will add an item to  replacement field in the kustomization file.
+The item must be either a file, or an inline string.
```

Comment:
From what I've seen, the inline string case is not supported by the code in this PR, so we should update the long description to match what the command does:
```suggestion
		Long: `this command will add an item to replacement field in the kustomization file.
The item must be a file path.
```

Reply:
updated. sorry for late reply

Reply:
"either" was still leftover 😅 I left a comment with that suggestion.

Reply:
sorry about that 😓 updated